collections/VideoCollection.js

```diff
@@ -0,0 +1,20 @@
+var VideoCollection = Backbone.Collection.extend({
+  url: function () {
+    return 'https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&type=video&videoEmbeddable=true&key=AIzaSyBSrRAvBkcFpsMUPTnM9EmITdLf_gG_9CI&q=' + appModel.attributes.searchQuery
```

Comment:
Again, we don't want to access attributes direction - that's what `get` and `set` are for. If we were to make changes on the `attributes` object directly, none of our change event triggers would fire.